CMSgov/onemac
# OY2-9714: State User can only Respond to RAI for packages that are in RAI Issued Status in OneMAC

Story: https://qmacbis.atlassian.net/browse/OY2-9714
Endpoint: https://d3vt6497msprdf.cloudfront.net

### Details
Modify Submission Dashboard to only display the “Respond to RAI” action for packages in the "RAI Issued" status.

### Changes
- changed the RAI popup inclusion condition from "not Withdrawn" to "must be RAI Issued"

### Implementation Notes
- Withdraw action has multiple possible statuses.  If we add more than one for "Respond to RAI," should mirror that logic.
- This is a logical place to add state machine behavior
- Cypress tests are failing on finding #transmittalNumberColHeader, which I can't find in the code and the test does not reference it in the master branch, so I feel like this might be a test that is being worked on, maybe for testing search and filter?  I've decided to stop letting the red Cypress Tests keep me from starting the PR.

### Test Plan
1.  Login in as a submitter user
2. access the Package Dashboard
3. click the actions icon for a package NOT in RAI Issued - should not have the "Respond to RAI" choice
4. click the actions icon for a package with status of "RAI Issued" - should see the "Respond to RAI" menu option.
5. Click "Respond to RAI" and submit the RAI Response as normal
6. Check the Submission Dashboard for the new submission
7. Verify the Package Dashboard behaves as before.
